__tests__/internal/move-after-handle-request/route-handlers/function-handler/basic-test.js

```diff
@@ -123,4 +136,47 @@ describe("Integration | Route handlers | Function handler", () => {
 
     expect(data).toEqual([{ id: "1" }, { id: "2" }, { id: "3" }]);
   });
+
+  test("it can serialize empty includes", async () => {
+    expect.assertions(1);
+
+    server.create("post", {
+      author: server.create("author", { id: 1, name: "Daniel" }),
+      title: "abcd"
+    });
+    server.create("post", {
+      author: server.create("author", { id: 2, name: "Alexandre" }),
+      title: "abcd"
+    });
+
+    server.get("/posts/:id", (schema, request) => {
+      return schema.posts.find(request.params.id);
+    });
+
+    let res = await fetch("/posts/1?include=");
+    expect(res.status).toBe(200);
+  });
+
+  test("it can serialize includes with trailing comma", async () => {
+    expect.assertions(2);
+
+    server.create("post", {
+      author: server.create("author", { id: 1, name: "Daniel" }),
+      title: "abcd"
+    });
+    server.create("post", {
+      author: server.create("author", { id: 2, name: "Alexandre" }),
+      title: "abcd"
+    });
+
+    server.get("/posts/:id", (schema, request) => {
+      return schema.posts.find(request.params.id);
+    });
+
+    let res = await fetch("/posts/1?include=author,");
+    expect(res.status).toBe(200);
+    expect(JSON.parse(res._bodyInit).included).toEqual([
+      { type: "authors", id: "1", attributes: { name: "Daniel" } }
+    ]);
```

Comment:
Can we follow the pattern from the other tests, and use `let data = await res.json()` here (`res._bodyInit` looks like a private method so we want to avoid that) and also assert against the shape of the data? Just enough to make sure the right data is there.

Reply:
Sorry for the delay, I was in other projects and Christmas came... Happy new year!!
Well, regarding your feedback, I took away the lower case test and I added the data expectation to the high level one.
In the high level test, I did also follow the pattern  from the other tests.
I hope it's all good! 
Cheers